llama_stack/providers/remote/telemetry/opentelemetry/opentelemetry.py

```diff
@@ -5,6 +5,9 @@
 # the root directory of this source tree.
```

Comment:
add the ability to specify the expected output through a generic API in telemetry

Reply:
will do this in a follow up PR.